### DISCUSSION TITLE
Problem with RNFastImage

### DISCUSSION THREAD

**Discussion by gopherios** [codemagic-ci-cd/codemagic-docs#968] [Q&A] [answered]
Hi everyone! I have some problem with build step - "xcode-project use-profiles"

My project include "react-native-fast-image". When i start building i have next error:

```
Failed to set code signing settings for node_modules/react-native-fast-image/ios/FastImage.xcodeproj
```
Someone solved this problem?

**Comment by himesh-cm** [comment, marked as answer]
Hi @AlexArtisan, what version of RN are you using?
Please check this SO [link](https://stackoverflow.com/a/64152729) for a possible solution.

**Reply by gopherios** [reply]
Thanks you for respond, i use RN 0.63.2. But this solution not for me. RNFastImage work fine and project build normal with my Xcode but not with CodeMagic.

**Reply by himesh-cm** [reply]
Can you please share the build id?

**Reply by gopherios** [reply]
Yes, 63298fa05ef2b16f10b5fac9

**Reply by himesh-cm** [reply]
It seems .xcodeproj for the FastImage is being code signed instead of the actual project.

can you please use the below change to your xcode-project use-profile command?

`xcode-project use-profiles --project ios/*.xcodeproj`

re-run the build, and let us know how it goes.

**Reply by gopherios** [reply]
Awesome! All worked! Tested with "TestFlight".
